### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Pavlo-lab-max-Kopalko**
https://Pavlo-lab-max-Kopalko.github.io/layout_catalog/
https://Pavlo-lab-max-Kopalko.github.io/layout_catalog/report/html_report/

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve 
It is necessary that the standard styles are also applied smoothly as the styles on hover, also try to fix all problems with tests, you need to check whether all styles match those on the design
image

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,180 @@
...
+
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border: 1px solid transparent;
+    text-decoration: none;
+
+    &:hover {
+      border-color: var(--blue-accent);
+      background-color: #fff;
```
Comment (loralevitska) [inline src/styles/main.scss:142]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

File: src/styles/moyo.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+:root {
+  --active-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
```
Comment (loralevitska) [inline src/styles/moyo.scss:13]: Move each BEM block to separate file

File: src/styles/stars.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
```
Comment (loralevitska) [inline src/styles/stars.scss:8]: Combine with shorthand `background` property

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,180 @@
...
+}
+
+.card {
+  padding-left: 16px;
+  padding-right: 16px;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  background-color: var(--white-accent);
```
[the remark this answers is not in the document]
Comment (Pavlo-lab-max-Kopalko) [reply src/styles/main.scss:51]: to regret I couldn't resolve the problems with test, therefore I'm asking you to do that. Mentors couldn't help me with it. Sometimes something like that occurs.

File: src/index.html [hunk 5]
```diff
@@ -5,8 +5,345 @@
...
+
+          <li class="nav__item">
+            <a href="#SMARTPHONES" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#LAPTOPS_&_COMPUTERS"
+              class="nav__link"
+              data-qa="nav-hover">Laptops & computers
```
Comment (volodymyr-soltys97) [inline src/index.html:41]:
```suggestion
            <a
              href="#LAPTOPS_&_COMPUTERS"
              class="nav__link"
              data-qa="nav-hover"
            >
              Laptops & computers
```

File: src/index.html [hunk 6]
```diff
@@ -5,8 +5,345 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#home" class="is-active nav__link">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#SAMSUNG" class="nav__link">Samsung</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:30]:
```suggestion
            <a href="#samsung" class="nav__link">Samsung</a>
```

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,180 @@
...
+    color: var(--main-accent);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    width: 166px;
+
+    &:hover {
+      color: #34568b;
+      // transition: transform 300ms;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:70]: Remove comments

File: src/styles/main.scss (hunk 4 above)
Comment (volodymyr-soltys97) [inline src/styles/main.scss:51]:
```suggestion
  border: 1px solid #f3f3f3;
  background-color: var(--white-accent);
  transition: transform 300ms;
```

File: src/styles/stars.scss (hunk 3 above)
Comment (volodymyr-soltys97) [reply src/styles/stars.scss:8]: Still not fixed from the previous review

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Still not fixed from the previous review
image

**Pull request closed without merging**